sdk/communication/azure-communication-identity/src/test/java/com/azure/communication/identity/CommunicationIdentityTests.java

```diff
@@ -320,12 +350,38 @@ public void revokeTokenWithResponseNullContext() {
     }
 
     @Test
-    public void getToken() {
+    public void getTokenWithChatScope() {
```

Comment:
The same comment that I left for `CommunicationIdentityAsyncTests.createUserAndTokenWithChatScope`

Reply:
Done